packages/libs/error-stack/src/report.rs

```diff
@@ -193,18 +193,16 @@ use crate::{
 #[must_use]
 #[repr(transparent)]
 pub struct Report<C> {
-    pub(super) frames: Vec<Frame>,
+    // reason: This implies a memory footprint equal to a single pointer size instead of three
```

Comment:
Generally we use `reason = ` within the actual attribute macro. Have we just written a normal comment here because we don't want a huge macro?

Reply:
`expect` and `reason` is currently only possible on nightly, thus I put the comment here 🙁

Reply:
Oh I didn't realise reason was nightly only too. Cheers!